app/client/components/helpCentre/helpCentreArticle.tsx

```diff
@@ -0,0 +1,131 @@
+import { navigate, RouteComponentProps } from "@reach/router";
+import * as Sentry from "@sentry/browser";
+import React, { useEffect, useState } from "react";
+import { SectionContent } from "../sectionContent";
+import { SectionHeader } from "../sectionHeader";
+import { Spinner } from "../spinner";
+import { WithStandardTopMargin } from "../WithStandardTopMargin";
+import { BackToHelpCentreButton } from "./BackToHelpCentreButton";
+import { helpCentreNavConfig } from "./helpCentreConfig";
+import { h2Css } from "./helpCentreStyles";
+import {
+  Article,
+  BaseNode,
+  ElementNode,
+  LinkNode,
+  TextNode
+} from "./HelpCentreTypes";
+
+interface HelpCentreArticleProps extends RouteComponentProps {
+  articleCode?: string;
+}
+
+const HelpCentreTopic = (props: HelpCentreArticleProps) => {
+  const [article, setArticle] = useState<Article | undefined>(undefined);
+
+  useEffect(() => {
+    setArticle(undefined);
+    fetch(`/api/help-centre/article/${props.articleCode}`)
+      .then(response => {
+        if (response.ok) {
+          return response.json();
+        } else {
+          Sentry.captureMessage(
+            `Fetching article ${props.articleCode} returned ${response.status}.`
+          );
+          navigate("/help-centre");
+        }
+      })
+      .then(articleData => setArticle(articleData as Article))
+      .catch(error =>
+        Sentry.captureException(
+          `Failed to fetch article ${props.articleCode}. Error: ${error}`
+        )
+      );
+  }, [props.articleCode]);
+
+  const selectedNavTopic = helpCentreNavConfig.find(
+    topic => topic.id === article?.topics[0].path
+  );
+
+  return (
+    <>
+      <SectionHeader title="How can we help you?" pageHasNav={true} />
+      <SectionContent hasNav={true} selectedTopicObject={selectedNavTopic}>
+        <h2 css={h2Css}>{article?.title}</h2>
+        {article ? <ArticleBody article={article} /> : <Loading />}
+        <BackToHelpCentreButton />
+      </SectionContent>
+    </>
+  );
+};
+
+const Loading = () => (
+  <WithStandardTopMargin>
+    <Spinner loadingMessage={"Fetching article..."} />
+  </WithStandardTopMargin>
+);
+
+// This is to appease React's "Lists need a unique key" error
+// A 5 character random string should be safe enough (no collisions)
+const getRandomKey = () =>
```

Comment:
Maybe we could replace this with some sort of unique counter?
eg
`hiThere1` or `hiThere2` or `hello1`